ee/app/license/server/methods.ts

```diff
@@ -1,12 +1,17 @@
 import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
 
-import { hasLicense } from './license';
+import { hasLicense, getModules } from './license';
 
 Meteor.methods({
-	'license:hasLicense'(feature: string) {
-		check(feature, String);
+	'license:hasLicense'(features: string[]) {
+		check(features, [String]);
 
-		return hasLicense(feature);
+		return features.map((feature) => ({
```

Comment:
we changing the return... show we use Array.some or Array.every, other case hasLicense will return always !== null/false

Reply:
This method isn't used anymore, but I forgot to revert the changes, I'll do